.github/workflows/docker/compose/3rd_parties-compose.yaml

```diff
@@ -6,3 +6,12 @@ services:
     build:
       dockerfile: comps/3rd_parties/nginx/src/Dockerfile
     image: ${REGISTRY:-opea}/nginx:${TAG:-latest}
+
+  wav2lip:
+    build:
+      dockerfile: comps/3rd_parties/wav2lip/Dockerfile
```

Comment:
In 3rd_parties, please also follow the same structure including: src, deployment/docker_compose, etc.